client_it_test.go

```diff
@@ -179,6 +180,67 @@ func TestSetRowAttrs(t *testing.T) {
 	}
 }
 
+func TestOrmCount(t *testing.T) {
+	client := getClient()
+	countFrame, err := index.Frame("count-test", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = client.EnsureFrame(countFrame)
+	if err != nil {
+		t.Fatal(err)
+	}
+	qry := index.BatchQuery(
+		countFrame.SetBit(10, 20),
+		countFrame.SetBit(10, 21),
+		countFrame.SetBit(15, 25),
+	)
+	client.Query(qry, nil)
+	response, err := client.Query(index.Count(countFrame.Bitmap(10)), nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if response.Result().Count != 2 {
+		t.Fatalf("Count should be 2")
+	}
+}
+
+func TestTopNReturns(t *testing.T) {
+	client := getClient()
+	frame, err := index.Frame("topn_test", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = client.EnsureFrame(frame)
+	if err != nil {
+		t.Fatal(err)
+	}
+	qry := index.BatchQuery(
+		frame.SetBit(10, 5),
+		frame.SetBit(10, 10),
+		frame.SetBit(10, 15),
+		frame.SetBit(20, 5),
+		frame.SetBit(30, 5),
+	)
+	client.Query(qry, nil)
+	time.Sleep(10 * time.Second)
```

Comment:
It might be a good idea to include a comment describing why this sleep is here. We might also consider exposing a way for the client to request a cache invalidation, so in that case the test could call that instead of sleeping.